website/views/shared/tasks/task_view/skills.jade

```diff
@@ -8,16 +8,26 @@ ul.items.rewards
 
 // Class Skills
 ul.items(ng-if='main && list.type=="reward" && user.stats.class && !user.preferences.disableClasses')
-  li.task.reward-item(ng-repeat='(k,skill) in Content.spells[user.stats.class]', ng-if='user.stats.lvl >= skill.lvl',popover-trigger='mouseenter', popover-placement='top', popover='{{skill.notes()}}')
+  li.task.reward-item.list-group-item(ng-repeat='(k,skill) in Content.spells[user.stats.class]', ng-if='user.stats.lvl >= skill.lvl',popover-trigger='mouseenter', popover-placement='top',
+  popover='{{skill.notes()}}{{k=="frost" && user.stats.buffs.streaks ? " You have already cast this today. Your streaks are frozen, and there\'s no need to cast this again." : ""}}}{{k=="stealth" ? " Number of dailies avoided: " + user.stats.buffs.stealth + "." : ""}}{{k=="stealth" && user.stats.buffs.stealth >= user.dailys.length ? " You have already avoided all your dailies, there\'s no need to cast this again." : ""}}')
```

Comment:
These should be made into translatable strings:

```
"You have already cast this today. Your streaks are frozen, and there\'s no need to cast this again."
" Number of dailies avoided: <%= number %>."
" You have already avoided all your dailies, there\'s no need to cast this again."
```

I understand that this spaghetti code is only here until we can actually do this work on the backend, but can we at least extract these pieces into Jade variables so it's easier to read?